docs/user/README.md

```diff
@@ -34,3 +34,11 @@ FlowForge supports deploying projects to remote _Devices_. They run a software a
 that connects back to the platform to receive updates.
 
  - [Working with Devices](devices.md)
+
+
+## Working with Files and Context
+
+FlowForge supports reading and writing persistent files and persistent context in the cloud.
```

Comment:
Not always FlowForge Cloud though. 

```suggestion
FlowForge supports reading and writing persistent files and persistent context. Files and context written can thus be retrieved after for example a Stack upgrade.
```

Reply:
```suggestion
FlowForge supports reading and writing persistent files and persistent context in your Node-RED project wherever it runs.
```

Counter proposal.

Not keen on adding info like "when you upgrade your stack" as it dates very quickly.

Maybe we could add from V1.2 onwards?

Reply:
@Steve-Mcl Each stack update you'll benefit? As a user it feels like I click the upgrade stack button each month or so?

Reply:
```suggestion
FlowForge supports reading and writing persistent files and persistent context in your Node-RED project wherever it runs.  This provides the benefit of data surviving a project restart, stack upgrade and other scenarios.
```

Better?

Reply:
outdated